manifest.ts

```diff
@@ -8,7 +8,7 @@ export default Manifest({
   name: "Announcement Bot",
   description: "Send an announcement to one or more channels",
   icon: "assets/icon.png",
-  outgoingDomains: ["cdn.skypack.dev"],
```

Comment:
🔏 This is a big unlock for enterprise developers with the most strict requirements!